data/hai missions.txt

```diff
@@ -141,6 +143,7 @@ mission "Ask the Hai about the Unfettered"
 	to offer
 		has "First Contact: Unfettered: offered"
 	on offer
+		log "Spoke to the Hai about their side of the Unfettered's story. If they are to be believed, the ancient Hai lost much of their territory to infighting, and since learned to live in stability and balance."
```

Comment:
I prefer `their side of the story of the Hai` of those two, I think. I kind of liked the idea of making the two log entries sound a little parallel, which is why I wanted "If they are to be believed" in both of them.